README.md

```diff
@@ -0,0 +1,128 @@
+## Scalar DL Node Client SDK
+
+This is a library for Node.js applications by which the applications can interact with [Scalar DL](https://github.com/scalar-labs/scalardl) network.
+
+## Install
+
+We can use any NPM package managers to install this library. For example, install with NPM:
+```
+npm install @scalar-labs/scalardl-node-client-sdk
+```
+
+## HOWTO
+
+### Create ClientService instance
+
+`ClientService` class is the main class of this package.
+It provides following functions to request Scalar DL network.
+
+|Name|use|
+|----|---|
+|registerCertificate|To register a client's certificate to a Scalar DL network|
+|registerContract|To register the contracts to the registered client of the Scalar DL network|
+|listContracts|To list all registered contracts of the client|
+|executeContract|To execute a registered contract of the client|
+|validateLedger|To validate an asset of the Scalar DL network to determine if it is tampered|
+
+Use the code snippet below to create a ClientService instance.
+
+```
+const { ClientService } = require('@scalar-labs/scalardl-web-client-sdk');
+const clientService = new ClientService(clientProperties);
+```
+
+The argument `clientProperties` object is mandatory for the constructor.
+This is an properties example that a user `foo@example.com` tries to connect to the server `scalardl.example.com:50051` of the Scalar DL network.
+```
+{
+    'scalar.ledger.client.server_host': 'scalardl.example.com',
+    'scalar.ledger.client.server_port': 50051,
+    'scalar.ledger.client.cert_holder_id': 'foo@example.com',
+    'scalar.ledger.client.private_key_pem': "-----BEGIN EC PRIVATE KEY-----\nMHc...",
+    'scalar.ledger.client.cert_pem': "-----BEGIN CERTIFICATE-----\nMIICjTCCAj...n",
+    'scalar.ledger.client.cert_version': 1,
+    'scalar.ledger.client.tls.enabled': false,
+    'scalar.ledger.client.tls.ca_root_cert': '// not used here',
+}
+```
+
+### Register the certificate
+When you have a clientService instance.
```

Comment:
I think this line should be deleted here and moved up. So something like:

> In what follows assume that we have a `clientService` instance.
> 
> ### Register the certificate
> 
> ...

Reply:
Updated